examples/DDG4_MySensDet/CMakeLists.txt

```diff
@@ -11,15 +11,14 @@
 cmake_minimum_required(VERSION 3.3 FATAL_ERROR)
```

Comment:
I might be good to set everywhere cmake minimum version to 3.12 and use `SET CMP0074 NEW` globally

Reply:
If we set cmake_minimum to 3.12 the policy will be set to new anyway.

Reply:
I don't think so in this case.

> This policy was introduced in CMake version 3.12. CMake version 3.12.4 warns when the policy is not set and uses OLD behavior. Use the cmake_policy() command to set it to OLD or NEW explicitly.

And the same behavior of warn when not set is also present in the last version (3.15).

Reply:
https://cmake.org/cmake/help/v3.12/manual/cmake-policies.7.html#introduction


> The cmake_minimum_required() command does more than report an error if a too-old >
> version of CMake is used to build a project. It also sets all policies introduced in that CMake 
> version or earlier to NEW behavior. To manage policies without increasing the minimum 
> required CMake version,